nextcloud-jail.sh

```diff
@@ -192,6 +197,40 @@ fi
 iocage fstab -a "${JAIL_NAME}" "${CONFIGS_PATH}" /mnt/configs nullfs rw 0 0
 iocage exec "${JAIL_NAME}" chown -R www:www /mnt/files
 iocage exec "${JAIL_NAME}" chmod -R 770 /mnt/files
+
+
+#####
+#
+# Dependency installation
+#
+#####
+
+if [ "${DATABASE}" = "mariadb" ]; then
+  iocage exec "${JAIL_NAME}" pkg install -qy mariadb103-server php73-pdo_mysql php73-mysqli
+elif [ "${DATABASE}" = "pgsql" ]; then
+  iocage exec "${JAIL_NAME}" pkg install -qy postgresql10-server php73-pgsql php73-pdo_pgsql
+fi
+
+# Create the jail, pre-installing needed packages
+cat <<__EOF__ >/tmp/pkg.json
+	{
+  "pkgs":[
+  "nano","sudo","redis","php73-ctype","gnupg","bash",
+  "php73-dom","php73-gd","php73-iconv","php73-json","php73-mbstring",
+  "php73-posix","php73-simplexml","php73-xmlreader","php73-xmlwriter",
+  "php73-zip","php73-zlib","php73-hash","php73-xml","php73","php73-pecl-redis",
+  "php73-session","php73-wddx","php73-xsl","php73-filter","php73-pecl-APCu",
+  "php73-curl","php73-fileinfo","php73-bz2","php73-intl","php73-openssl",
+  "php73-ldap","php73-ftp","php73-imap","php73-exif","php73-gmp",
+  "php73-pecl-memcache","php73-pecl-imagick","bash","perl5",
+  "p5-Locale-gettext","help2man","texinfo","m4","autoconf"
+  ]
+}
+__EOF__
+
```

Comment:
Moving this block to after the jail creation (which happens on line 155 of your version) means that none of these packages will be installed when the jail is created (and thus, none of them will be installed at all).  Creating `/tmp/pkg.json` needs to be done before jail creation; installing ports in the jail can only be done once the jail has been created.

Reply:
Whoops I I shouldn't have moved this block indeed... my mistake...
will fix... 

(Should've have been in this PR actually... was already fixed locally)

Reply:
Fixed.
Sorry for the inconvenience!